go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/percona/pmm
 
-go 1.21
+go 1.22
+
+toolchain go1.22.2
```

Comment:
We can drop the toolchain directive - less stuff to update on every new patch version release.

Reply:
we can't, you can see that CI failed because of not having that. By the last commit I had to re-add it back.

Reply:
I suppose this is because we need to have the same Go version inside the image/container (i.e. perconalab/pmm-server:dev-latest), which is not the case.

Reply:
IMO, if we bump it up here, we have to use the same version everywhere, including the build tools, like `rpmbuild`.

Reply:
great part about it that our build system started using go1.22 after merging this PR and nothing is broken.